lib/components/Button/Button.types.ts

```diff
@@ -4,7 +4,7 @@ export interface ButtonProps
   extends React.ButtonHTMLAttributes<HTMLButtonElement> {
   variant?: "primary" | "destructive";
   buttonStyle?: "primary" | "secondary" | "tertiary" | "outlined";
-  buttonSize?: "sXmall" | "small" | "medium" | "large";
+  buttonSize?: "xSmall" | "small" | "medium" | "large";
```

Comment:
nice catch!
please mark this commit with the BREAKING CHANGE footer, because this changes the Button API.

`BREAKING CHANGE: changes button API size options`

Reply:
done